src/app/plugins/pubsub.plugin.ts

```diff
@@ -125,13 +109,24 @@ export class PubSubPlugin extends Plugin {
     return embed;
   }
 
+  private _parseSub(subData: any): subData {
```

Comment:
if the `subData` type (btw, please call it `SubData` because it shouldnt be lower camelcase) matches the JSON returned by API, you shouldnt have to do any "parse" or post-processing

Reply:
Doesn't match it completely, it has last_sale_end which isn't in the original one.

Reply:
correction 2 items change status is received as a string, but I have it so it saves as a boolean so it looks better with if statements.